### PR TITLE
Release 0.6 fixes

### PR DESCRIPTION
**Pull request by DanChirtoaca**

### CODE REVIEW

File: src/umr/static/styles/articles.css [hunk 1]
```diff
@@ -163,6 +163,11 @@ a
    color: black !important;
 }
 
+#return_home
+{
+  color: rgba(0,0,0,.26) !important;
```
Comment (Lukeslux) [inline src/umr/static/styles/articles.css]: color is #030303

Comment (DanChirtoaca) [inline src/umr/static/styles/articles.css]: this is the mdl gray color that we have for undo and copy toggle, #303030 is practically black, is that what Mircea wanted?

Comment (Lukeslux) [inline src/umr/static/styles/articles.css]: Used [this tool](https://chrome.google.com/webstore/detail/colorzilla/bhlhnicpbhignbdhedgjhgdocnmhomnp/related), as the css for the burger button was difficult to find. It seems the be that color, does it look different to you>

Comment (Lukeslux) [inline src/umr/static/styles/articles.css]: > could be as gray as the burger menu on the left
Is what he wanted, but maybe making them as gray and the undo and copy is better...

Comment (DanChirtoaca) [inline src/umr/static/styles/articles.css]: this gray looks fine - check it out!

**Pull request merged**